retropie-shell-script-boilerplate.sh

```diff
@@ -1,33 +1,45 @@
 #!/usr/bin/env bash
 
-# [SCRIPT_TITLE] (e.g. Retropie Shell Script Boilerplate.)
```

Comment:
I prefer to call it [SCRIPT_TITLE] because it's the title (h1) of the README as well. And also because [SCRIPT_TITLE] != [SCRIPT_NAME] (e.g. [SCRIPT_TITLE] = RetroPie Shell Script Boilerplate,  [SCRIPT_NAME] = retropie-shell-script-boilerplate).

Maybe we could have:
```
# [SCRIPT_NAME] (e.g.: retropie-shell-script-boilerplate.sh)

# [SCRIPT_TITLE] (e.g. Retropie Shell Script Boilerplate.)
# [SCRIPT_DESCRIPTION] (e.g. A template for building shell scripts for RetroPie.)
```

Reply:
having the actual script filename right below the shebang is common practice between shell programmers. So if you really want to have `[SCRIPT_TITLE]` using your suggestion putting the `[SCRIPT_NAME]` at the top seems to be fine.

in my scripts I usually do
```
#!/usr/bin/env bash
# script-filename.sh
# A tool to let you make blablabla
```

I'm just thinking that all of this stuff on this repo is a bit overkill for a script template, but you're the boss here. :-)

Reply:
I'm not the boss of anyone nor anything ;) We are here working together! :D
But yeah, I think a like better to have a mix with you ideas and mine.

Can we agree on?

```
#!/usr/bin/env bash
# [SCRIPT_NAME].sh
#
# [SCRIPT_TITLE]
# [SCRIPT_DESCRIPTION]
```

Reply:
sure! Fully agreed!